waltid-services/waltid-wallet-api/src/main/kotlin/id/walt/webwallet/service/account/AccountsService.kt

```diff
@@ -29,7 +29,10 @@ object AccountsService {
                 WalletServiceManager.createWallet(tenant, registeredUserId)
             }.also { walletId ->
                 //TODO: inject
-                WalletServiceManager.issuerUseCase.add(IssuerDataTransferObject.default(walletId))
+                val issuer = IssuerDataTransferObject.default(walletId)
+                if (issuer != null) {
```

Comment:
you can use the safe call operator (?.) to simplify the null check and avoid the explicit if statement

Reply:
Code updated, please check now.